### PR TITLE
feat: Add new theme `LightContrasted` and apply it when necessary

### PR DESCRIPTION
**Pull request by zatteo**
When the user log in a Cozy instance, we want to apply the correct
theme based on the instance configuration but also on the user's
preferences

The instance configuration is defined by the
`passwords.theme.default-contrasted`. If this flag is true, then the
default theme should be the `LightContrasted` one. Otherwise the
`System` theme should be applied

The user's preferences are defined when the user changes the app theme
from the settings page. From that point we want to keep the selected
theme even when the user logs out

If the user did not change the app theme from the settings page, then a
log out should revert to the `LightContrasted` theme, which is the
app's default theme

Note that if the instance's flag is true, then we want to enforce the
`LightContrasted` whatever if the user did set a different theme or not
in their previous session

___

This PR also update the `Light` theme latest Cozy's latest color palettes

### CODE REVIEW

File: apps/browser/src/background/main.background.ts [hunk 1]
```diff
@@ -820,6 +821,14 @@ export default class MainBackground {
   async logout(expired: boolean, userId?: string) {
     await this.eventUploadService.uploadEvents(userId);
 
+    // Cozy customization, reset theme to LighContrasted if user did not manually changed it
+    //*
+    const isUserSetTheme = await this.stateService.getIsUserSetTheme();
+    if (!isUserSetTheme) {
+      await this.stateService.setTheme(ThemeType.LightContrasted);
+    }
```
Comment (zatteo) [inline apps/browser/src/background/main.background.ts:829]: In which user case does this happen ?

**Comment by zatteo** [comment]
Approved ✅ 

(can't officially approve because I created the PR)

File: apps/browser/src/background/main.background.ts (hunk 1 above)
Comment (Ldoppea) [reply apps/browser/src/background/main.background.ts:829]: This corresponds to the following case:
> If the user did not change the app theme from the settings page, then a
log out should revert to the LightContrasted theme, which is the
app's default theme

Comment (Ldoppea) [reply apps/browser/src/background/main.background.ts:829]: So the most common scenario will be:
- app starts with default contrasted theme
- user logs in a Cozy that does not enforce contrasted theme, so app will automatically switch to classic light theme
- on logout, app will revert to contrasted theme

**Pull request merged**